modules/acm/src/main/java/edu/gemini/epics/acm/CaCommandSender.java

```diff
@@ -13,7 +13,7 @@
  * @author jluhrs
  *
  */
-public interface CaCommandSender {
+public interface CaCommandSender extends CaCommandTrigger {
```

Comment:
What is the difference between an Apply and a Command, if you don't mind me asking?

Reply:
That is related to Gemini EPICS records. A command is a record (CAD) that has inputs, and can be triggered. An Apply is a record that has no parameters, but connects to one or more CAD records. It can be triggered, and it will trigger the connected CADs in sequence. Another difference is that the Apply record can produce the CLID, the number that identifies one specific instance of execution. Both CADs and Applies can be associated to a CAR record that publishes the execution state of the CAD or Apply.

Reply:
Thanks very much for that great explanation! I really appreciate it.